ST_indivs.c

```diff
@@ -111,11 +112,32 @@ Bool indiv_New( SppIndex sp) {
   p->Prev = NULL;
   Species[sp]->IndvHead = p;
 
-  //sql for inserting new indiv
-  //if(!UseGrid)
-	//  insertIndiv(p);
   id++;
-  return( TRUE);
+  return TRUE;
+}
+
+/* Copy one individual's information to another individual. 
+   Note: this does not modify either individual's linked list functionality.
+   Both individuals MUST be allocated. */
```

Comment:
@kpalmqui `copy_individual` does copy the values directly. However, it is only used to copy the spinup conditions to a new location for running the simulation. 

I needed a way to "freeze" the entire grid following spinup so it could be loaded multiple times, one for each iteration of the simulation. The easiest way was to do something like `GridCells ** spinupConditions = gridCells` then, when we want to load the spinup conditions we allocate gridCells again and call the copy functions like `copy_individual` to move the data over. 

The data must be copied to a new location in memory or the first simulation iteration would overwrite the spinup values and there would be no way to recover them for the second iteration.

Reply:
@chaukap thanks